insights/parsers/awx_manage.py

```diff
@@ -34,6 +36,28 @@ class AnsibleTowerLicenseType(CommandParser, JSONParser):
     def parse_content(self, content):
```

Comment:
Please refer to the following example to deprecate an existing parser:

https://github.com/RedHatInsights/insights-core/blob/41eaa85a0a393e707e488f7354af0ecec9eb89dc/insights/parsers/aws_instance_type.py#L46-L48